Makefile

```diff
@@ -263,6 +280,10 @@ lint/templates: specinstall
 	$(SPECTRAL) lint templates/*.yml templates/*.yaml --ignore-unknown-format --ruleset .validate-templates.yaml
 .PHONY: lint/templates
 
+MARKDOWN_LINK_CHECK_CONFIG_DIR ?= $(PROJECT_PATH)/.markdown-link-check
+MARKDOWN_LINK_CHECK_CONFIG ?= $(MARKDOWN_LINK_CHECK_CONFIG_DIR)/markdown-link-check-config.json
+lint/markdown-link-check: markdown-link-check-install
```

Comment:
This hasn't been added to the `lint` task intentionally as it might be time consuming I kept it separate from there.